core/tests/test_floor_sections.model.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from rest_framework.exceptions import ValidationError
+
+from ..models import OfficeBlock, FloorSection
+
+from core.tests import CoreBaseTestCase
+
+User = get_user_model()
+
+
+class OfficeBlockModelTest(CoreBaseTestCase):
```

Comment:
To keep the tests DRY, move the test cases specific to an office section to `core/tests/test_office.py`.

Reply:
Move the tests cases to `core/tests/test_office.py` and delete this module. Also, refactor the `test_office.py` module to keep it DRY. Case in point; `setUp` has some repetitive statements.
https://github.com/AndelaOSP/art-backend/blob/bbf6724bc1c7c08e31113d6921852c834c4d8e34/core/tests/test_office.py#L17-L20

https://github.com/AndelaOSP/art-backend/blob/bbf6724bc1c7c08e31113d6921852c834c4d8e34/core/tests/test_office.py#L75-L78